deploy/roles/apache/tasks/main.yml

```diff
@@ -51,3 +49,20 @@
 - name: Save rules
   command: "/sbin/service iptables save"
   when: iptables_config
+
+- name: create certs for apache localhost
+  command: openssl req -x509 -nodes -days 365 -newkey rsa:2048 -subj "/C=US/ST=CFPB/L=Washington/O=Dis/CN=www.cf.gov" -keyout /etc/pki/tls/private/localhost.key -out /etc/pki/tls/certs/localhost.crt
+  args:
+    creates: /etc/pki/tls/certs/localhost.crt
+  when: not custom_repo
+
+- name: Allow admins to read logs
+  acl:
+    path: "{{ item }}"
+    entity: "{{ restricted_group_name }}"
+    etype: group
+    permissions: rx
+    state: present
+  with_items:
+    - /var/log/httpd24/
+    - /var/log/httpd/
```

Comment:
Also, could you try `rX` for the permissions?  In the normal setfacl command, using a capital X instead of lowercase x will only make directories executable, and not files.  It would be nice if this works in Ansible too, but it's not the end of the world if log files end up being executable.